README.md

```diff
@@ -1,3 +1,7 @@
+[![Build Status](https://travis-ci.org/andela/healthchecks-apollo.svg?branch=develop)](https://travis-ci.org/andela/healthchecks-apollo)
+[![Coverage Status](https://coveralls.io/repos/github/andela/healthchecks-apollo/badge.svg?branch=master)](https://coveralls.io/github/andela/healthchecks-apollo?branch=develop)
```

Comment:
Can you change this to point to the correct branch

Reply:
Getting right on it